src/BlockEntities/BrewingstandEntity.h

```diff
@@ -119,6 +126,9 @@ class cBrewingstandEntity :
 	/** Amount of ticks that the current item has been brewed */
 	short m_TimeBrewed;
 
+	/** The fuel for the brewing stand */
+	short m_Fuel;
```

Comment:
Fuel in brewing stands doesn't work by ticks; it's decremented each time brewing starts (but once it starts, the brewing of that batch of potions will continue without decrementing it further).